lib/scope.js

```diff
@@ -1125,28 +1113,87 @@ module.exports = {
+        // probably add a confusing message to the report, so mutating seems, sadly, more useful.
+        var_datum.value = `False`;  // The original value is still preserved
+        // Warn the developer not to use `.there_is_another`.
+        await scope.addToReport( scope, { key: `id`, value: `WARNING: The attribute \`.there_is_another\` is invalid in story table tests. Replace it with \`.target_number\` in your \`var\` and \`trigger\` columns. Set the \`value\` to the number of items in that list. This test will now set this row's \`value\` to \`False\`. See https://suffolklitlab.github.io/docassemble-AssemblyLine-documentation/docs/automated_integrated_testing#there_is_another. The row data is\n${ JSON.stringify( var_dataum.original )}` });
+      }
+    }
+
+    return enhanced_var_data;
+  },  // Ends scope.ensureSpecialRows()
+
+  getThereIsAnotherValue: async function ( scope, { row, from_table=true }) {
+    /* Given story row-like data as well as whether this is from a table row,
+    * return the best guess as to a safe value to set for this round. Give
+    * the developer a warning if
     * it's an inappropriate value. Avoid mutation.
     *
     * WARNING: Remember to decrement the actual value later.
+    * 
+    * Sometimes this will be from `.target_number` from story tables and
+    * we believe we've set it up so that people will make it a number.
+    * Sometimes this will be from `.there_is_another`. We think we've ensured
+    * that it will only come from Steps and should be `True` or `False`.
+    * This feels precarious, but I can't see another permuation.
+    * 
+    * This may in future be used to normalize all values that need `True`/`False`
+    * normalization.
     */
-    if ( parseInt( value ) < 1 || value === `False` || value === `false` ) {  // Loop will end
+    let value = row.value;
+    let int_val = parseInt( value );
+    
+    // Step stuff. Take into account a lot of possible values
+    if ( value.match( /(false|no|unchecked|uncheck|deselected|deselect)/i )) {
       return `False`;
 
-    } else if ( !from_table && ( value === `True` || value === `true` )) {
-      // This is not a table row, it's a step or something, so we should be safe from infinite loops
-      return `True`;
+    } else if ( value.match( /(true|yes|checked|check|selected|select)/i )) {
+      if ( from_table ) {
+        // Just in case, but this should never happen.
+        await scope.addToReport( scope, { key: `id`, value: `WARNING: The attribute \`.there_is_another\` is invalid in story table tests. Replace it with \`.target_number\` in your \`var\` and \`trigger\` columns. Set the \`value\` to the number of items in that list. This test will now set this row's \`value\` to \`False\`. See https://suffolklitlab.github.io/docassemble-AssemblyLine-documentation/docs/automated_integrated_testing#there_is_another. The row data is\n${ JSON.stringify( row.original )}` });
```

Comment:
Is this the right error message here? I think someone could reach here even if they don't have `there_is_another` in their table, e.g., `| x.target_number | true | users.target_number |`

Also, the linked documentation seems to indicate that `there_is_another` is valid in the trigger column. This error message says otherwise.

Reply:
re `| x.target_number | true |` - good catch. That's invalid input too. I'm not sure how to handle that exactly. This might be 'good enough for now' + creating an issue for thinking about that. I'd love your thoughts. Made #296.

re the trigger column: Nice catch there too. That definitely can be changed right away.